db/const.txt

```diff
@@ -668,9 +669,11 @@ RC_Demon	6
 RC_DemiHuman	7
 RC_Angel	8
 RC_Dragon	9
-RC_Boss	10
-RC_NonBoss	11
-RC_NonDemiHuman	12
+RC_Player	10
+RC_Boss	11
+RC_NonBoss	12
+RC_NonDemiHuman	13
```

Comment:
That's just my assumption (that's the only thing that made demihumans special, compared to the other races)